frontend/src/metabase/collections/components/CollectionHeader/CollectionHeader.tsx

```diff
@@ -32,6 +38,9 @@ const CollectionHeader = ({
         onUpdateCollection={onUpdateCollection}
       />
       <HeaderActions data-testid="collection-menu">
+        {isDataAppCollection(collection) && (
+          <LaunchDataAppButton collection={collection} />
+        )}
```

Comment:
We're going to introduce a dedicated data app "landing" page, so this change is temporal